mscore/prefsdialog.ui

```diff
@@ -3453,6 +3453,49 @@ Adjusting latency can help synchronize your MIDI hardware with MuseScore's inter
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="resetPositionsOnMuseScore1And2Import">
+         <property name="title">
+          <string>Reset Positions on MuseScore 1 and 2 Import</string>
+         </property>
+         <layout class="QHBoxLayout" name="horizontalLayout_7">
+          <item>
+           <widget class="QRadioButton" name="resetPositionsAlwaysAsk">
```

Comment:
I think the “Always ask” option should come last (after “Yes” and “No”), mirroring the layout in the prompt dialog.

Reply:
In that dialog the "don't ask" comes first